app/validation/validate_schema_text.py

```diff
@@ -0,0 +1,49 @@
+import re
+from jsonpointer import resolve_pointer
+from eq_translations.utils import find_pointers_to
+from eq_translations.survey_schema import SurveySchema
+
+
+def get_pointer_keys():
+    pointer_keys = [
+        'show_guidance',
+        'hide_guidance',
+        'description',
+        'legal_basis',
+        'playback',
+        'item_title',
+        'label',
+        'add_link_text',
+        'empty_list_text',
+        'text',
+        'title',
+    ]
+
+    return pointer_keys
+
+
+def get_list_pointer_contents(schema, schema_elements):
+    contents = []
+    for element in schema_elements:
+        pointer_contents = resolve_pointer(schema, element)
+        contents.append(pointer_contents)
+    return contents
+
+
+def validate_json_schema_text(json_schema):
+    pointers = []
+    errors = []
+
+    for key in get_pointer_keys():
+        pointers.extend(find_pointers_to(json_schema, key))
+
+    schema_object = SurveySchema()
+    schema_object.schema = json_schema
+    pointers.extend(schema_object.get_list_pointers())
+
+    for pointer in pointers:
+        schema_text = resolve_pointer(json_schema, pointer)
+        if re.search(r"'", schema_text):
```

Comment:
The regex should be compiled before the for loop, so it's not done on every iteration:

```python
quote_regex = re.compile(r"'")
for pointer in pointers:
    schema_text = resolve_pointer(json_schema, pointer)
    if quote_regex.search(schema_text):
```

Also, the regex should also search for `"`.

Reply:
Fixed this